src/main/haskell/kore/src/Kore/Logger.hs

```diff
@@ -0,0 +1,128 @@
+{-|
+Module      : Kore.Logger
+Description : Logging functions.
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+Maintainer  : vladimir.ciobanu@runtimeverification.com
+Stability   : experimental
```

Comment:
For this and all the other new modules, I would be inclined to leave off `Stability` and `Portability`. `Stability` doesn't mean anything, except it makes a tiny note in the upper right-hand corner of the generated documentation that nobody looks at. All of our `Portability` fields are wrong, anyway: they should say `GHC-only`, which is what *everyone's* say, or should say, so it's just noise.